docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst

```diff
@@ -30,75 +30,75 @@ For this, you'd like to run an Apache Flink job and write the filtered messages
 
 ``services.tf`` file:
 
-.. code:: terraform
+.. code:: bash
 
   resource "aiven_flink" "flink" {
     project      = var.project_name
     cloud_name   = "google-europe-west1"
     plan         = "business-8"
     service_name = "demo-flink"
   }
-
+  
   resource "aiven_kafka" "kafka" {
     project      = var.project_name
     cloud_name   = "google-europe-west1"
     plan         = "business-8"
     service_name = "demo-kafka"
   }
-
+  
   resource "aiven_service_integration" "flink_to_kafka" {
     project                  = var.project_name
     integration_type         = "flink"
     destination_service_name = aiven_flink.flink.service_name
     source_service_name      = aiven_kafka.kafka.service_name
   }
-
+  
   resource "aiven_kafka_topic" "source" {
     project      = var.project_name
     service_name = aiven_kafka.kafka.service_name
     partitions   = 2
     replication  = 3
     topic_name   = "iot_measurements"
   }
-
+  
   resource "aiven_kafka_topic" "sink" {
     project      = var.project_name
     service_name = aiven_kafka.kafka.service_name
     partitions   = 2
     replication  = 3
     topic_name   = "cpu_high_usage"
   }
-
+  
   resource "aiven_flink_table" "source" {
     project        = var.project_name
     service_name   = aiven_flink.flink.service_name
     integration_id = aiven_service_integration.flink_to_kafka.integration_id
     table_name     = "iot_measurements_table"
     kafka_topic    = aiven_kafka_topic.source.topic_name
-    schema_sql = <<EOF
-      hostname STRING,
-      cpu STRING,
-      usage DOUBLE,
-      occurred_at BIGINT,
-      time_ltz AS TO_TIMESTAMP_LTZ(occurred_at, 3),
-      WATERMARK FOR time_ltz AS time_ltz - INTERVAL '10' SECOND
-    EOF
+    schema_sql     = <<EOF
```

Comment:
please try "<<-EOF"; pygments has a bug parsing heredoc comments: https://github.com/pygments/pygments/issues/2162

Reply:
stripping the leading whitespace from the closing EOF is the proper way towrite it, technically pygments is spec conform there!